exercises/concept/tim-from-marketing/.meta/design.md

```diff
@@ -18,3 +18,17 @@
 
 - `strings`: strings will be compared to null and basic methods from strings will be called.
 - `if-else-statements`: using a conditional statement.
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `essential`: If the student did not use the `null` expression in the `print` method, instruct the student to do so.
```

Comment:
Note that this comment currently would still apply to the example solution you linked to in the PR. 

To be honest I think we can remove this comment completely now that we have the one covering optionals: I don't think there is a way to pass the tests without using either `null` or `java.util.Optional`.

Reply:
👍